src/lessons/02_introduction_to_frontend/introduction-to-frontend.slides.html

```diff
@@ -0,0 +1,688 @@
+              <img data-src="images/vscode-folder-opened.png" class="fragment r-frame current-visible" data-fragment-index="4"/>
+            </div>
+
+            <aside class="notes">
+              We ain't gonna need a terminal right now but it's good to have it opened for
+            </aside>
+          </section>
+        </section>
+
+        <!--SLIDE 7-->
+        <!--Inspecting elements (demo)-->
+        <section>
+          <!--SLIDE 7.1-->
+          <section data-auto-animate data-timing="10">
+            <h3>Now it's time to inspect elements all together! ❤️</h3>
+            <p class="small-please" style="font-style: italic;">demo being presented</p>
+          </section>
+        </section>
+
+        <!--SLIDE 8-->
+        <!--Other tags-->
+        <section>
+          <!--SLIDE 8.1-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+          </section>
+
+          <!--SLIDE 8.2-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+            <p class="small-please">Let's learn a few other tags in HTML which will be important in the upcoming lessons and topics</p>
+
+            <aside class="notes">
+              Explain what means "structural", in this case. (Compare with head)
+            </aside>
+          </section>
+
+          <!--SLIDE 8.3-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+            <p class="small-please">Let's learn a few other tags in HTML which will be important in the upcoming lessons and topics</p>
+
+            <ul class="tags-with-code">
+              <li><code>&lt;div&gt;</code> defines a division or a section in an HTML document.</li>
+              <li><code>&lt;img&gt;</code> used to embed an image in an HTML page. <small style="vertical-align: middle;">(remember: there's not content within img)</small></li>
+              <li><code>&lt;a&gt;</code> defines a hyperlink, which is used to link from one page to another. (`a` as in anchor)</li>
+              <li class="fragment small-please" style="margin-top: 50px; color: gray;"><code>&lt;!-- SOME COMMENT --&gt;</code> specifies a comment in a page (only visible for developers).</li>
+            </ul>
+
+            <div class="fragment">
+              <p>👇🏼 The ones you know already</p>
+
+              <ul class="tags-with-code">
+                <li><code>&lt;p&gt;</code> represents a paragraph.</li>
+                <li><code>&lt;h1&gt;, &lt;h2&gt; ... &lt;h6&gt;</code> used to define HTML headings, from most to less important.</li>
+              </ul>
+            </div>
+          </section>
+
+          <!--SLIDE 8.4-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+            <p class="small-please"><b>Remind</b>: Some of those tags contain important attributes!</p>
```

Comment:
I think you meant **Reminder** in here :P and the next ones as well

Reply:
Tnxxx